### PR TITLE
Fixing busted build with node v10 and refactoring and fixing busted flow spec unit tests

### PR DESCRIPTION
**Pull request by john-burns-arundo**
This allows yarn to install properly with node v10, and should also support better upwards compatibility. Also fixes build error on latest with some improperly escaped tests in flow spec. Includes refactoring flow spec unit tests to prevent tripping the refactor checker.

### CODE REVIEW

**Comment by knsv** [comment]
Hi! Sorry for the slow handling. We are trying to expand the team and pickup the pace! If you still want to help with the project, please get the latest code and resolve the conflicts. This would be much appreciated.

If you are not interested or unable to do the update, the PR will be closed 60 days from now.
👋

File: package.json [hunk 1]
```diff
@@ -37,36 +38,37 @@
...
-    "jest": "^22.4.2",
-    "jison": "^0.4.18",
-    "node-sass": "^4.7.2",
-    "sass-loader": "^6.0.7",
-    "standard": "^11.0.1",
-    "webpack": "^4.1.1",
-    "webpack-cli": "^2.0.12",
-    "webpack-node-externals": "^1.6.0",
-    "yarn-upgrade-all": "^0.3.0"
+    "babel-core": ">=6.26.0",
```
Comment (barakplasma) [inline package.json]: Why do you want the devDependencies to be "greater than or equal to" instead of "compatible with"?

**Comment by chris579** [comment]
@knsv This pr can be closed, the 60 days are way over.

**Comment by Dunning-Kruger** [comment]
I'll go ahead and declare this one stale / expired manually to keep the PR list clean. Sorry for any inconvenience.

**Pull request closed without merging**